recipes/osx-pocl-opencl/meta.yaml

```diff
@@ -0,0 +1,22 @@
+package:
+  name: osx-pocl-opencl
+  version: 1
+
+build:
+  number: 0
+  track_features:
+    - osx_pocl_opencl
+  skip: True  # [not osx]
+
+test: {}
+
+about:
+  home: http://github.com/conda-forge/osx-pocl-opencl-feedstock
+  license: BSD 3-clause
+  license_family: BSD
+  summary: Metapackage to track whether OpenCL on OSX is being provided by POCL (rather than the OS)
+
+extra:
+  recipe-maintainers:
+    - inducer
+    - mattwala
```

Comment:
Oh ok. As we don't generally know people's affiliations (work or otherwise) and they do change periodically, we have tended to just ask to be sure. If certain people will often appear on a recipes one proposes, we ask that they "signoff" on being generally included by a certain person as a maintainer in issue ( https://github.com/conda-forge/staged-recipes/issues/139 ) if the chose to so as to reduce redundant requests. This makes it easy for other reviewers to confirm this as well as it is in a central location.